### PR TITLE
data concept: add implementation draft

### PR DESCRIPTION
**Pull request by d70-t**
This PR adds first thoughts on the implementation of the data concept.

### CODE REVIEW

File: orcestra_book/data_concept.md [hunk 1]
```diff
@@ -104,3 +104,117 @@ e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)
...
+:::{card} [STAC](https://stacspec.org/)
+
+SpatioTemporal Asset Catalogs, the STAC specification is a common language to describe geospatial information.
+
+* ✅ stable format
+* ✅ [integrations for multiple languages](https://stacspec.org/en/about/tools-resources/) exist
+* ✅ can be used with Intake
+* ✅ common set of earth observation related metadata is defined
+* ❌ more complicated to create (but tools exist)
+* ❌ can only be used for
```
Comment (lkluft) [inline orcestra_book/data_concept.md:141]: for what? ;)

File: orcestra_book/data_concept.md [hunk 2]
```diff
@@ -104,3 +104,117 @@ e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)
...
+* ✅ stable format
+* ✅ [integrations for multiple languages](https://stacspec.org/en/about/tools-resources/) exist
+* ✅ can be used with Intake
+* ✅ common set of earth observation related metadata is defined
+* ❌ more complicated to create (but tools exist)
+* ❌ can only be used for 
+:::
+::::
+
+In any case, the catalog should be accessible though a well-known public URL, such that users always know where to start.
```
Comment (lkluft) [inline orcestra_book/data_concept.md:145]: *through

File: orcestra_book/data_concept.md [hunk 3]
```diff
@@ -104,3 +104,117 @@ e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)
...
+
+* ✅ HTTP as access protocol
+* ✅ compatible with about everything
+* ❌ prone to [link rot](https://en.wikipedia.org/wiki/Link_rot)
+* ❌ single point of failure
+* ❌ in general: no guarantees about data integrity or persistence
+:::
+
+:::{card} DOI repo
+This includes e.g. [Pangea](https://pangaea.de), [Aeris](https://www.aeris-data.fr), [Zenodo](https://zenodo.org), etc...
```
Comment (lkluft) [inline orcestra_book/data_concept.md:168]: Typo in pang**a**ea

**Review by lkluft — COMMENTED** [review]
Looks nice, thanks! 👌

if the CI passes, we are good to merge

**Comment by d70-t** [comment]
Thanks for the review!
I hope I could fix the CI.

**Pull request merged**